block-storage-attacher/Makefile

```diff
@@ -68,31 +77,27 @@ coverage:
 
 
 .PHONY: buildimage
-buildimage: build-systemutil
```

Comment:
Create a new make target called buildimage-fips for the current changes and leave the existing changes as it is.